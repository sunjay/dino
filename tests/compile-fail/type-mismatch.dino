fn main() {
    let x: int = 10j;
    x = b"abc";
}
