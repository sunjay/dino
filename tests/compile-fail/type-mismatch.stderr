Error: In 'tests/compile-fail/type-mismatch.dino': mismatched types
